platform/platform.go

```diff
@@ -14,7 +14,7 @@ import (
 var websockets []chan string
 var runUploadLoop = false
 
-func AddWebsocket() chan string {
+func addWebsocket() chan string {
```

Comment:
Why are all these public ? We are calling them only in the `platform` package.

Reply:
This is question for you, you wrote this code. I made it private right in this PR.